src/main/java/kuromi/Ui.java

```diff
@@ -0,0 +1,66 @@
+package kuromi;
+
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user (e.g. reading inputs, showing outputs).
+ */
+public class Ui {
```

Comment:
Good job on following the naming convention of your methods in the Ui class as per the coding standard!